### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by BenagainB**
my web app deployment location: https://purple-meadow-03f38911e.1.azurestaticapps.net/

### CODE REVIEW

File: wordle-web/pages/index.vue [hunk 1]
```diff
@@ -7,9 +7,9 @@
       </v-card>
       <v-card>
         <v-card-title class="headline">
-          Welcome to the Vuetify + Nuxt.js template
+          Welcome to the NEW Vuetify + Nuxt.js template
```
Comment (hewrtarkhany) [inline wordle-web/pages/index.vue]: nice catch

File: .github/workflows/azure-static-web-apps-CICD.yml [hunk 2]
```diff
@@ -36,6 +36,7 @@ jobs:
         id: builddeploy
         uses: Azure/static-web-apps-deploy@v1
         with:
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}
```
Comment (hewrtarkhany) [inline .github/workflows/azure-static-web-apps-CICD.yml:39]: nice catch on catching the secret token

File: .github/workflows/azure-static-web-apps-purple-meadow-03f38911e.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Build And Deploy
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v1
```
Comment (hewrtarkhany) [inline .github/workflows/azure-static-web-apps-purple-meadow-03f38911e.yml:37]: this needed the dependency, awesome job figuring it out

File: .github/workflows/azure-static-web-apps-CICD.yml [hunk 4]
```diff
@@ -54,5 +54,5 @@ jobs:
         id: closepullrequest
         uses: Azure/static-web-apps-deploy@v1
         with:
-          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_CALM_MEADOW_06C89A91E }}
```
Comment (hewrtarkhany) [inline .github/workflows/azure-static-web-apps-CICD.yml]: awesome job catching those secrets

**Review by hewrtarkhany — APPROVED** [review]

File: .github/workflows/azure-static-web-apps-purple-meadow-03f38911e.yml (hunk 3 above)
Comment (BenagainB) [reply .github/workflows/azure-static-web-apps-purple-meadow-03f38911e.yml:37]: I learned from the best

File: wordle-web/pages/game.vue [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        <v-col>
+            <v-card>
+                <h1>Wordles with Turtles</h1>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+                <p> Yertle was a turtle who knew oh so many wordles. As a young turtle, he was a bit of a nerdle (he even wore a girtle!). Now he's a bit of a flirtle, though his looks make milk curdle, and his favorite scent is myrtle and his favorite color is purple. He calls his home a yurtle!</p>
+            </v-card>
+            <v-card-actions>
+                <v-btn color="primary" nuxt to="/"> Back to Main </v-btn>
+            </v-card-actions>
```
Comment (kelseymcmahon) [inline wordle-web/pages/game.vue:12]: This section seems to be missing some <v-card-text> tags to help with the padding issues on the card. Would also recommend using <the v-card-title> for the header.

**Comment by JessicaH46** [comment]
Your buttons and everything looked good to me! Really liked the Yertle the wordle turtle. Good job!

File: wordle-web/pages/game.vue (hunk 5 above)
Comment (BenagainB) [reply wordle-web/pages/game.vue:12]: Thank you. I'm adjusting for the next assignment.

**Comment by mmwoodfo** [comment]
In the future, just make sure that you are making a separate branch for your assignment, rather than making a PR using your main. Code changes showing up here in the PR should only be relevant to the assignment the PR is for.

Here is an example of a PR that uses an Assignment 1 branch: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/11

Great job!

**Comment by mmwoodfo** [comment]
Did you have a partner for this?

**Pull request closed without merging**